src/examples/mc_pos_control_multiplatform/mc_pos_control.cpp

```diff
@@ -930,12 +932,14 @@ void  MulticopterPositionControlMultiplatform::handle_vehicle_attitude(const px4
 						_R(i, 2) = body_z(i);
 					}
 
-					/* copy rotation matrix to attitude setpoint topic */
-					memcpy(&_att_sp_msg.data().R_body[0], _R.data, sizeof(_att_sp_msg.data().R_body));
-					_att_sp_msg.data().R_valid = true;
-
 					/* calculate euler angles, for logging only, must not be used for control */
 					math::Vector<3> eul = _R.to_euler();
+
```

Comment:
move above euler angles computation such that calculation and assignment are one block